src/database-controller/test-case.md

```diff
@@ -0,0 +1,146 @@
+# Database Controller Test Cases
+
```

Comment:
Can your tests cover all operations/lines in graph [https://github.com/microsoft/pai/issues/4651](https://github.com/microsoft/pai/issues/4651)?

Reply:
Not all of #4651, you can refer to this graph here. I tested every path in it:

![image](https://user-images.githubusercontent.com/7499023/89498358-ad6dd280-d7f0-11ea-83ae-4f419ee2e871.png)